pkg/cli/config/config.go

```diff
@@ -85,6 +85,10 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	bpConfig.SessionDirectory = viper.GetString("session-dir")
 	bpConfig.AssumeInitialArn = viper.GetString("assume-initial-arn")
 
+	if bpConfig.AssumeInitialArn == "" {
```

Comment:
nit: please add test case to improve the coverage rate
```suggestion
	if bpConfig.AssumeInitialArn == "" {
```
![image](https://github.com/openshift/backplane-cli/assets/5784943/c8ed1b11-97ba-474e-8f7a-582d08b41f24)

Reply:
Also, didn't we implement to detect `assume-initial-arn` from OCM URL? 
Looks it has been hard-coded: 
https://github.com/openshift/backplane-cli/blob/30def33dca39dc676d1c8675a7972f995c7afdb0/cmd/ocm-backplane/cloud/common.go#L223-L231

May I know why we still need to check in backplane's configuration for that arn?

Reply:
As I understand, This is not the right way to get the assume-role ARN for the OCM env, It should get it via bp-api/ocm API endpoints, not hardcode in the bp-cli :( 

This is a public repo, and this exposes all bp-api URL's and role ARN's 

FYI : @bmeng / @supreeth7 / @cblecker

Reply:
I agree, exposing that OCM API endpoint could lead to unauthorized access attempts which could be risky. 

Also, exposing an ARN for an AWS IAM role is riskier since it's unique identifiers for AWS resources, and while an arn alone doesn't grant access without corresponding credentials, it can provide useful information to an unauthorized user which can still have potential risks.